Aptoide/aptoide-client-v8
# [DT-468] recent popular search

**What does this PR do?**

This PR aims at fixing the recent and popular search views, to meet the new requirements with the popular apps

**Database changed?**

   Yes 

**Where should the reviewer start?**

- [ ] SearchUseCase.kt

**How should this be manually tested?**

  Open app go to search and check results and the flow between buzz suggestions and search history

**What are the relevant tickets?**

  Tickets related to this pull-request: [DT-468](https://aptoide.atlassian.net/browse/DT-468)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**


[DT-468]: https://aptoide.atlassian.net/browse/DT-468?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
